src/main/java/com/uvocab/api/service/AuthService.java

```diff
@@ -0,0 +1,29 @@
+package com.uvocab.api.service;
+
+import lombok.AllArgsConstructor;
+import org.springframework.security.authentication.AuthenticationManager;
+import org.springframework.security.authentication.UsernamePasswordAuthenticationToken;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Service;
+import uvocab.protobuf.v1.User;
+
+@Service
+@AllArgsConstructor
+public class AuthService {
+
+  private final AuthenticationManager authenticationManager;
+
+  private static final String FIXED_TOKEN = "str_token";
+
+  public String login(User user) {
+    Authentication authentication =
+        authenticationManager.authenticate(
+            new UsernamePasswordAuthenticationToken(user.getLogin(), user.getPassword()));
```

Comment:
@Danilosimoes as I said in this [comment](https://github.com/robsonoduarte/u-vocab-api/pull/65#discussion_r1702144722) the **AuthenticationManager** do  all we need in this moment ( _authenticate the user or throw a exception if user/password in not valid_ ), then all we need is call the method **authenticate** and return the token.